### PR TITLE
[formrecognizer] README and release updates 

### PR DESCRIPTION
**Pull request by catalinaperalta**
Updating the README to work with the current preview. Adding links to the service's models page for new prebuilts. 
Ran black on the code, committing all changes except those in _models.py

### CODE REVIEW

File: sdk/formrecognizer/azure-ai-formrecognizer/README.md [hunk 1]
```diff
@@ -158,7 +158,7 @@ document_analysis_client = DocumentAnalysisClient(
 
 ### DocumentAnalysisClient
 `DocumentAnalysisClient` provides operations for analyzing input documents using prebuilt and custom models through the `begin_analyze_document` and `begin_analyze_document_from_url` APIs.
-Use the `model` parameter to select the type of model for analysis.
+Use the `model` parameter to select the type of model for analysis. The following table includes some of the models supported by the service (for a full list of supported models, see [here][fr-models]):
 
 |Model| Features
```
Comment (kristapratico) [inline sdk/formrecognizer/azure-ai-formrecognizer/README.md:163]: wondering if it's better to just remove the table and the link out to the service docs. It's a nice visual, but I worry people might not read and miss the "includes some of the models". Just a thought, not blocking this PR

**Review by kristapratico — APPROVED** [review]

File: sdk/formrecognizer/azure-ai-formrecognizer/README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Azure Form Recognizer client library for Python
 
-Azure Cognitive Services Form Recognizer is a cloud service that uses machine learning to analyze text and structured data from your documents. It includes the following main features:
+Azure Form Recognizer is a cloud service that uses machine learning to analyze text and structured data from your documents. It includes the following main features:
```
[the remark this answers is not in the document]
Comment (catalinaperalta) [reply sdk/formrecognizer/azure-ai-formrecognizer/README.md:3]: I'm waiting for us all to discuss that during scrum today. No decision has been made yet

File: sdk/formrecognizer/azure-ai-formrecognizer/README.md (hunk 1 above)
Comment (catalinaperalta) [reply sdk/formrecognizer/azure-ai-formrecognizer/README.md:163]: Hmmm that's a better idea. I'll remove it now

**Pull request merged**